ios/Runner/pigeon.m

```diff
@@ -21,98 +21,61 @@ static id GetNullableObjectAtIndex(NSArray *array, NSInteger key) {
   return (result == [NSNull null]) ? nil : result;
 }
 
-@interface Machine ()
```

Comment:
We should ignore this generated file under ios

Reply:
Removed from repo